x-pack/legacy/plugins/siem/cypress/integration/signal_detection_rules_prebuilt.spec.ts

```diff
@@ -55,3 +70,100 @@ describe('Signal detection rules, prebuilt rules', () => {
     });
   });
 });
+
+describe('Deleting prebuilt rules', () => {
+  beforeEach(() => {
+    esArchiverLoad('prebuilt_rules_loaded');
+    loginAndWaitForPageWithoutDateRange(DETECTIONS);
+    waitForSignalsPanelToBeLoaded();
+    waitForSignalsIndexToBeCreated();
+    goToManageSignalDetectionRules();
+  });
+
+  afterEach(() => {
+    esArchiverUnload('prebuilt_rules_loaded');
+  });
+
+  it('Does not allow to delete one rule when more than one is selected', () => {
+    const numberOfRulesToBeSelected = 2;
+    selectNumberOfRules(numberOfRulesToBeSelected);
+
+    cy.get(COLLAPSED_ACTION_BTN).each(collapsedItemActionBtn => {
+      cy.wrap(collapsedItemActionBtn).should('have.attr', 'disabled');
+    });
+  });
+
+  it('Deletes and recovers one rule', () => {
+    const expectedNumberOfRulesAfterDeletion = 91;
+    const expectedNumberOfRulesAfterRecovering = 92;
```

Comment:
While you [mentioned yesterday](https://github.com/elastic/kibana/pull/61059#discussion_r397298821) that we want to make sure we don't derive the total rules from test data in case there are any issues loading the pre-built rules, it might be useful from a maintenance perspective to pull this total out as an export so when additional rules are added (like will be happening in the next day or so) we'll only have to update it in one place. Thoughts?

Reply:
Thanks for the suggestion 😊